src/replays/replayitem.py

```diff
@@ -105,175 +90,194 @@ def __init__(self, uid, parent, *args, **kwargs):
+        self.mod       = message["mod"]
+
         # Map preview code
         self.mapdisplayname = maps.getDisplayName(self.mapname)
       
         self.icon = maps.preview(self.mapname)
         if not self.icon:
             self.client.downloader.downloadMap(self.mapname, self, True)
             self.icon = util.icon("games/unknown_map.png")        
-        #self.setIcon(0, self.icon)
-        
-        self.moddisplayname = self.mod
-        self.modoptions = []
 
-        if self.mod in mods :
+        if self.mod in mods:
             self.moddisplayname = mods[self.mod].name 
+        else:
+            self.moddisplayname = self.mod
 
 #        self.title      = message['title']
-#        self.host       = message['host']
 #        self.teams      = message['teams']
 #        self.access     = message.get('access', 'public')
 #        self.mod        = message['featured_mod']
+#        self.host       = message["host"]
 #        self.options    = message.get('options', [])
 #        self.numplayers = message.get('num_players', 0) 
 #        self.slots      = message.get('max_players',12)
 
-        self.viewtext = (self.FORMATTER_REPLAY.format(time=self.startHour, name=self.name, map = self.mapdisplayname, duration = self.duration, mod = self.moddisplayname))
+        self.viewtext = self.FORMATTER_REPLAY.format(time=self.startHour, name=self.name, map=self.mapdisplayname,
+                                                     duration=self.duration, mod=self.moddisplayname)
 
     def infoPlayers(self, players):
-        '''
-        processes information from the server about a replay into readable extra information for the user, also calls method to show the information
-        '''
+        # processes information from the server about a replay into readable extra information for the user, #
+        # also calls method to show the information #
+
         self.moreInfo = True
+        self.numberplayers = len(players)
         mvpscore = 0
         mvp = None
         scores = {}
-        
-        for player in players :
-            team            = int(player["team"])
-
-            if team : #even if it's FFA the players are still in team 1
-                if "score" in player :
-                    if player["score"] > mvpscore:
-                        mvp = player
-                        mvpscore = player["score"]
-
-                    if team in scores :
-                        scores[team] = scores[team] + player["score"]
-                    else :
-                        scores[team] = player["score"]
-                if not team in self.teams :
+
+        for player in players:  # player highscore
+            if "score" in player:
+                if player["score"] > mvpscore:
+                    mvp = player
+                    mvpscore = player["score"]
+
+        for player in players:  # player -> teams & playerscore -> teamscore
+            if self.mod == "phantomx" or self.mod == "murderparty":
```

Comment:
No I mean the hardcoding of the `phantomx` and `murderparty` mods. Don't do that.